Makefile

```diff
@@ -1,7 +1,15 @@
 VERSION:=$(shell cat VERSION)
 
+IMAGE_NAME ?= prometheus-example-app
```

Comment:
let's maybe name it with prefix (`rhobs/prometheus-example-app` or even `ghcr.io/rhobs/prometheus-example-app`), this way creating development images would be simpler

Reply:
It does get built with the full `ghcr.io` name when running make locally, but the way the default GitHuib Workflow is set up, this `IMAGE_NAME` variable is set to just the app / repo name.